ballista/rust/scheduler/src/main.rs

```diff
@@ -62,14 +63,18 @@ async fn start_server(
         "Ballista v{} Scheduler listening on {:?}",
         BALLISTA_VERSION, addr
     );
+    //there should be only one SchedulerServer in process
```

Comment:
I don't understand how this code ensures just one `SchedulerServer` exists per process.

It seems to me what this code does is create a new `SchedulerServer` on a port that the operating system picks.

Reply:
Perhaps using something like `lazy_static` (https://docs.rs/lazy_static/1.4.0/lazy_static/, already a dependency) might be worth looking into

Reply:
Thanks,  sorry for my wrong annotation. it should be `only call SchedulerServer::new() once in process ` ,
is this right ?

Reply:
its ok to `clone()` SchedulerServer in multi-thread without call `tokio::spawn(async move { state_clone.synchronize_job_status_loop().await });`

Reply:
SchedulerServer::new creates the scheduler loop that runs in the background, that's the thing that's causing the issue.

Reply:
@alamb Could you review it again, Thanks!

Reply:
Sorry -- I don't really know what this code is doing, so I just had an uninformed question; Sorry if this delayed merging unnecessarily.